web/e2e-tests/realm-creation.test.ts

```diff
@@ -78,6 +78,11 @@ async function realm_creation_tests(page: Page): Promise<void> {
     // element of id `lightbox_overlay` exists.
     await page.waitForSelector("#lightbox_overlay"); // if element doesn't exist,timeout error raises
 
+    // Check if the modal having the onboarding video has been displayed.
+    // Close it to avoid blocking further test execution.
+    await page.waitForSelector("#onboarding-video-modal", {visible: true});
+    await page.click("#onboarding-video-modal .modal__close");
+
```

Comment:
We might want to have `populate_db` mark all the `OnboardingSteps` for the initial onboarding things for most of the original users; maybe excluding a couple accounts (say, Aaron and Zoe); I feel like the "Welcome to inbox" type modals usually not helpful when doing development environment work, and that may apply here too.

Reply:
> We might want to have populate_db mark all the OnboardingSteps for the initial onboarding things for most of the original users;

Done

> maybe excluding a couple accounts (say, Aaron and Zoe)

I think the better approach is mark it as seen for everyone. When you want to test these onboarding steps, create a new user. Logically, it makes sense as onboarding steps are for new users.

There's a very low chance that someone will see the `populate_db` code to figure out that these two users have onboarding steps NOT marked as seen.